android/modules/ui/src/java/ti/modules/titanium/ui/android/AndroidModule.java

```diff
@@ -228,6 +229,13 @@
 	@Kroll.constant
 	public static final int OVER_SCROLL_NEVER = 2; //android.view.View.OVER_SCROLL_NEVER;
 
+	@Kroll.constant
+	public static final int MODE_NIGHT_NO = Configuration.UI_MODE_NIGHT_NO;
+	@Kroll.constant
+	public static final int MODE_NIGHT_YES = Configuration.UI_MODE_NIGHT_YES;
+	@Kroll.constant
+	public static final int MODE_NIGHT_UNDEFINED = Configuration.UI_MODE_NIGHT_UNDEFINED;
```

Comment:
Can you map `nightModeStatus` to the existing [https://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI-property-semanticColorType](`semanticColorType`) property and it's `Ti.UI.SEMANTIC_COLOR_TYPE_DARK` + `SEMANTIC_COLOR_TYPE_LIGHT` type?

Reply:
Those constants are used as values of `Ti.UI.Android.nightModeStatus` which is an Android analogue of `Titanium.App.iOS.userInterfaceStyle`.
I believe `Titanium.App.iOS.userInterfaceStyle` does not return `SEMANTIC_COLOR_TYPE_DARK` or `SEMANTIC_COLOR_TYPE_LIGHT, does it (never checked, docs says it is `String[]`)?

Reply:
It returns a Number (USER_INTERFACE_STYLE_UNSPECIFIED = 0, USER_INTERFACE_STYLE_LIGHT = 1, USER_INTERFACE_STYLE_DARK = 2). So if you change the docs, it should be all set.

Reply:
`USER_INTERFACE_STYLE_*` constants are defined in `Ti.App.iOS` namespace.